app/javascript/components/game/game.css

```diff
@@ -3,85 +3,84 @@ body {
   color: #fff;
   font-family: Helvetica;
   font-weight: bold;
-  text-align:center;
+  text-align: center;
   background-color: rgb(17, 17, 20);
 }
 
 h1 {
-    text-transform: uppercase;
-    letter-spacing: 0.1em;
-    color: #f1f1f1
-    
+  text-transform: uppercase;
+  letter-spacing: 0.1em;
+  color: #f1f1f1;
 }
 
 #game {
-    width: 302px;
-    height: 302px;
-    border: 1px solid #dadada;
-    margin: 0 auto;
-    padding: 0;
-    margin-bottom: 20px;
+  width: 302px;
+  height: 302px;
+  border: 1px solid #dadada;
+  margin: 0 auto;
+  padding: 0;
+  margin-bottom: 20px;
 }
 
 #game > li {
-    list-style: none;
-    float: left;
-    overflow: hidden;
-    text-decoration:none;
-    width: 100px;
-    height: 100px;
-    background: #f1f1f1;
-    border: 1px solid #ccc;
-    border-right: 1px solid #fff;
-    cursor: pointer;
-    text-transform: uppercase;
-    text-align: center;
-    padding-top: 20px;
-    -moz-box-sizing: border-box;
-    box-sizing: border-box;
+  list-style: none;
+  float: left;
+  overflow: hidden;
+  text-decoration: none;
+  width: 100px;
+  height: 100px;
+  background: #f1f1f1;
+  border: 1px solid #ccc;
+  border-right: 1px solid #fff;
+  cursor: pointer;
+  text-transform: uppercase;
+  text-align: center;
+  padding-top: 20px;
+  -moz-box-sizing: border-box;
+  box-sizing: border-box;
 }
 
 #game > li:hover {
-    background: #f9f9f9;
+  background: #f9f9f9;
 }
 
 #game > li:active {
-    width: 100px;
-    height: 100px;
-    border: 0;
+  width: 100px;
+  height: 100px;
+  border: 0;
 }
 
 #reset-game {
   text-align: center;
-      border: none;
-      padding: 0.6em 1.2em;
-      background: #ed4e6e;
-      color: #fff;
-      font-size: 0.7em;
-      letter-spacing: 1px;
-      text-transform: uppercase;
-      cursor: pointer;
-      display: inline-block;
-      margin: 3px 2px;
-      border-radius: 20px;
+  border: none;
+  padding: 0.6em 1.2em;
+  background: #ed4e6e;
+  color: #fff;
+  font-size: 0.7em;
+  letter-spacing: 1px;
+  text-transform: uppercase;
+  cursor: pointer;
+  display: inline-block;
+  margin: 3px 2px;
+  border-radius: 20px;
 }
 
 #reset-game:hover {
-      background: #2c3e52;
+  background: #2c3e52;
 }
 
 #players {
```

Comment:
While we can style an element with either a class or ID, I generally use class for styling because classes can be reused and ID implies that this element & style is unique and un-reusable.